src/Tools/ConfigurationSchemaGenerator/ConfigSchemaEmitter.cs

```diff
@@ -325,6 +328,29 @@ private static void AppendSpaceIfNecessary(StringBuilder builder, string value)
         }
     }
 
+    internal static void AppendUnindentedValue(StringBuilder builder, string value)
+    {
+        var index = 0;
+
+        foreach (var match in Indentation().EnumerateMatches(value))
+        {
+            if (match.Index > index)
```

Comment:
In what case is `match.Index <= index` possible?

Reply:
When the first match is at the beginning of the line.